README.md

```diff
@@ -3,6 +3,7 @@
 # CCExtractor
 
 [![Build Status](https://travis-ci.org/CCExtractor/ccextractor.svg?branch=master)](https://travis-ci.org/CCExtractor/ccextractor)
+[![SourceForge](https://img.shields.io/badge/SourceForge%20downloads-213k%2Ftotal-brightgreen.svg)](https://sourceforge.net/projects/ccextractor/)
```

Comment:
But this is a static image.. it's not update automatically, right?

Reply:
That doesn't matter if we remove the SF downloads anyway in favour of GH releases.

Reply:
@saurabhshri yes because otherwise when the releases will be deleted, an automatic counter would error.

_Sent from my Moto G (3rd Gen) using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_